sdk/python/foundation-models/system/inference/text-to-image/safe-image-text-to-image-online-endpoint.ipynb

```diff
@@ -0,0 +1,761 @@
+    "    aacs = aacs_client.accounts.get(resource_group, aacs_name)\n",
+    "    print(f\"Found existing AACS Account {aacs.name}.\")\n",
+    "except:\n",
+    "    try:\n",
+    "        # check if there is an existing AACS resource within same resource group\n",
+    "        aacs = find_acs(aacs_client.accounts.list_by_resource_group(resource_group))\n",
+    "        print(\n",
+    "            f\"Found existing AACS Account {aacs.name} in resource group {resource_group}.\"\n",
+    "        )\n",
+    "    except:\n",
+    "        print(f\"Creating AACS Account {aacs_name}.\")\n",
+    "        aacs_client.accounts.begin_create(resource_group, aacs_name, parameters).wait()\n",
+    "        print(\"Resource created.\")\n",
+    "        aacs = aacs_client.accounts.get(resource_group, aacs_name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "aacs_endpoint = aacs.properties.endpoint\n",
+    "aacs_resource_id = aacs.id\n",
+    "aacs_name = aacs.name\n",
+    "print(\n",
+    "    f\"AACS name is {aacs.name} .\\nUse this name in UAI preparation notebook to create UAI.\"\n",
+    ")\n",
+    "print(f\"AACS endpoint is {aacs_endpoint}\")\n",
+    "print(f\"AACS ResourceId is {aacs_resource_id}\")\n",
+    "\n",
+    "aacs_access_key = aacs_client.accounts.list_keys(\n",
+    "    resource_group_name=resource_group, account_name=aacs.name\n",
+    ").key1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 2.3 Check if UAI is used (Required for using UAI authentication method)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "uai_id = \"\"\n",
+    "uai_client_id = \"\"\n",
+    "if uai_name != \"\":\n",
+    "    from azure.mgmt.msi import ManagedServiceIdentityClient\n",
+    "    from azure.mgmt.msi.models import Identity\n",
+    "\n",
+    "    try:\n",
+    "        msi_client = ManagedServiceIdentityClient(\n",
+    "            subscription_id=subscription_id,\n",
+    "            credential=credential,\n",
+    "        )\n",
+    "        uai_resource = msi_client.user_assigned_identities.get(resource_group, uai_name)\n",
+    "        uai_id = uai_resource.id\n",
+    "        uai_client_id = uai_resource.client_id\n",
+    "    except Exception as ex:\n",
+    "        print(\"Please run aacs-prepare-uai.ipynb notebook and re-run the cell.\")\n",
+    "        raise ex"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Pick a model to deploy\n",
+    "\n",
+    "Browse models in the Model Catalog in the AzureML Studio, filtering by the `image-text-to-image` task. In this example, we use the `stabilityai-stable-diffusion-xl-refiner-1-0` model. If you have opened this notebook for a different model, replace the model name accordingly. This is a pre-trained model."
```

Comment:
Task name. Same comment as above.

The task should be image-to-image.
Have left the same comment on model asset PR - https://github.com/Azure/azureml-assets/pull/1785.